saleor/wishlist/error_codes.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class WishlistErrorCode(Enum):
```

Comment:
According to our Notion docs: 
```suggestion
class WishlistErrorCode(str, Enum):
```